heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -43,15 +48,23 @@ export interface IJestPluginOptions {
 
 export interface IHeftJestConfiguration extends Config.InitialOptions {}
 
+const PLUGIN_NAME: string = 'JestPlugin';
+const PLUGIN_SCHEMA_PATH: string = `${__dirname}/schemas/heft-jest-plugin.schema.json`;
+const JEST_CONFIGURATION_LOCATION: string = `config/jest.config.json`;
+const OWN_PACKAGE_FOLDER: string = path.resolve(__dirname, '..');
+
+const ROOTDIR_TOKEN: string = '<rootDir>';
+const CONFIGDIR_TOKEN: string = '<configDir>';
+const PACKAGE_CAPTUREGROUP: string = 'package';
+const PACKAGEDIR_REGEX: RegExp = new RegExp(`^<packageDir:\\s*(?<${PACKAGE_CAPTUREGROUP}>[^\\s^]*)\\s*>`);
```

Comment:
```suggestion
const PACKAGEDIR_REGEX: RegExp = /^<packageDir:\s*(?<package>[^\s^]*)\s*>/;
```
RegExp's are difficult enough to read already without `` ` `` `` ` `` escaping superimposed on them. 😁